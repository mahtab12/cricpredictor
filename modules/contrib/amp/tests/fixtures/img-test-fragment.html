<img src="https://upload.wikimedia.org/wikipedia/commons/8/8a/Carnegie_Hall_Tower.JPG" />
