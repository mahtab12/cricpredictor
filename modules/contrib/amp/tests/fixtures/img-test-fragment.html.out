<amp-img src="https://upload.wikimedia.org/wikipedia/commons/8/8a/Carnegie_Hall_Tower.JPG" width="1920" height="2560" layout="responsive"></amp-img>
